mne/beamformer/_rap_music.py

```diff
@@ -224,6 +224,10 @@ def rap_music(evoked, forward, noise_cov, n_dipoles=5, return_residual=False,
         The residual a.k.a. data not explained by the dipoles.
         Only returned if return_residual is True.
 
+    See Also
+    --------
+    mne.Dipole.fit_dipole
```

Comment:
can you get away with just `fit_dipole` here? If you can't, it should be `mne.dipole.fit_dipole` or probably just `mne.fit_dipole` (I think we import it there)